src/central_control/scripts/central_control.py

```diff
@@ -14,7 +14,7 @@ class CentralControl:
     def __init__(self):
         host_ip = rospy.get_param('/host_ip')
 
-        self.client = airsim.CarClient(ip = host_ip)
+        self.client = airsim.CarClient()
```

Comment:
Don't think this ip is needed for any of us, any it's getting annoying forgetting to remove it when checking out a new branch